services/notify/src/handler.js

```diff
@@ -132,6 +132,58 @@ class Handler {
             throw err;
           }
         }
+        case 'slack-channel': {
+          const channelId = route.slice(2, route.length - 1).join('.');
+
+          const emojiMap = {
```

Comment:
this would make @owlishDeveloper very happy to see I expect!

Reply:
I was wondering whether this might be considered a little out of place for the base implementation. I copied this from our old CI system and I find the emojis so good for noticing what's going on at a glance. I'm happy it seems to have gone down well! :)